notebooks/image_models/solutions/2_mnist_cnn.ipynb

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
<p>This is really nice! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/651/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>